ui/src/screen/project/source.ts

```diff
@@ -92,11 +92,12 @@ export const fetch = async (project: Project, peer: User): Promise<void> => {
       source.fetchCommits(project.urn, peer.peerId, selectedRevision),
       fetchTreeRoot(selectedRevision),
     ]);
+    const grouppedHistory = source.groupCommitHistory(history);
 
     screenStore.success({
       code: writable<Code>(root),
-      history,
-      menuItems: menuItems(project, history),
+      history: grouppedHistory,
```

Comment:
Sorry for not making that explicit, but the typo is also in this variable and further down in `grouppedCommits`

Reply:
Forgot to grep for lowercase too :see_no_evil: